distribution/bin/generate-binary-notice.py

```diff
@@ -57,7 +57,7 @@ def generate_notice(source_notice, dependences_yaml):
     # Print Apache license first.
     print_outfile(source_notice)
     with open(dependences_yaml, encoding='utf-8') as registry_file:
-        dependencies = list(yaml.load_all(registry_file))
+        dependencies = list(yaml.load_all(registry_file, Loader=yaml.Loader))
```

Comment:
Can you check your YAML version ?
Is this working with pyyaml==5.4.1 and pyyaml==6.0.0 ?

Reply:
@cryptoe Both pyyaml==5.4.1 and pyyaml==6.0.0 works on my hello world test.

```python
import yaml
print(yaml.load_all(open('hello.yaml', 'r'), Loader=yaml.Loader))
```